src/search/searchBar.component.tsx

```diff
@@ -265,24 +267,32 @@ const SearchBar = (props: SearchBarProps): React.ReactElement => {
 
   // Updates the search fields when a session is loaded
   React.useEffect(() => {
-    if (shotnumRange) {
-      setSearchParameterShotnumMax(shotnumRange.max);
-      setSearchParameterShotnumMin(shotnumRange.min);
+    if (dateRange.fromDate) {
+      setSearchParameterFromDate(new Date(dateRange.fromDate));
     }
+    if (dateRange.toDate) {
+      setSearchParameterToDate(new Date(dateRange.toDate));
+    }
+
     if (experimentID) {
       setSearchParameterExperiment(experimentID);
     }
 
-    if (dateRange.fromDate && dateRange.toDate) {
-      const reduxSessionFromDate = new Date(dateRange.fromDate);
-      const reduxSessionToDate = new Date(dateRange.toDate);
-      setSearchParameterFromDate(reduxSessionFromDate);
-      setSearchParameterToDate(reduxSessionToDate);
-    }
-    if (maxShotsParam) {
-      setMaxShots(maxShotsParam);
+    setMaxShots(maxShotsParam);
+  }, [dateRange, experimentID, maxShotsParam]);
+
+  // removes all the search fields when a session is loaded
+  React.useEffect(() => {
+    if (sessionId) {
+      setSearchParameterExperiment(null);
+      setSearchParameterFromDate(null);
+      setSearchParameterToDate(null);
+      setSearchParameterShotnumMin(undefined);
+      setSearchParameterShotnumMax(undefined);
+      setMaxShots(MAX_SHOTS_VALUES[0]);
     }
-  }, [dateRange, experimentID, maxShotsParam, shotnumRange]);
+  }, [sessionId]);
```

Comment:
This code causes a bug where if a user navigates to the `Plots` tab and navigates back - it clears all the search fields. We probably need to not run the hook on mount to avoid this - so probably use a ref to keep track of the first render and not run this code on first render.